Formula/tarsnap-gui.rb

```diff
@@ -13,17 +14,41 @@ class TarsnapGui < Formula
     sha256 cellar: :any, high_sierra: "d3e5e2aeee094f0e5944d68765965f2b9261cce026bafe45482a2b1e9abb1273"
   end
 
-  depends_on "qt"
+  depends_on "qt@5"
   depends_on "tarsnap"
 
+  # Work around build error: Set: Entry, ":CFBundleGetInfoString", Does Not Exist
+  # Issue ref: https://github.com/Tarsnap/tarsnap-gui/issues/557
+  patch :DATA
+
   def install
     system "qmake"
     system "make"
-    system "macdeployqt", "Tarsnap.app"
```

Comment:
Looks like this got added back in #16130.

This is similar to CMake's `fixup_bundle` and it makes the app bundle standalone resulting in a bundled Qt5. This is why we never detected incompatibility with Qt6 as the `qt` dependency wasn't used.